### PR TITLE
Scrubbed all files replacing for JMS acronym

### PR DESCRIPTION
**Pull request by rmonson**
Replaced references to JMS in API files with Jakarta Messaging except where used in package and type names. Also modified POMs and Readme.adoc files.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,15 +23,15 @@
...
 
-    <groupId>org.eclipse.ee4j.jms</groupId>
-    <artifactId>jms-parent</artifactId>
+    <groupId>jakarta.messaging</groupId>
+    <artifactId>jakarta.messsaging-parent</artifactId>
     <packaging>pom</packaging>
-    <version>2.1-SNAPSHOT</version>
-    <name>jms</name>
-    <description>jms</description>
+    <version>2.2-SNAPSHOT</version>
```
Comment (bshannon) [inline pom.xml]: This is still version 2.1 of the API.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Eclipse Project for JMS
+# Eclipse Project for Jakarta Messaging
```
Comment (bshannon) [inline README.md]: Remove "Eclipse Project for".

File: api/pom.xml [hunk 3]
```diff
@@ -21,15 +21,28 @@
...
-        <version>1.0.4</version>
+        <version>1.0.5/version>
     </parent>
 
-    <groupId>jakarta.jms</groupId>
-    <artifactId>jakarta.jms-api</artifactId>
-    <version>2.1-SNAPSHOT</version>
+    <groupId>jakarta.messaging</groupId>
+    <artifactId>jakarta.messaging-api</artifactId>
+    <version>2.2-SNAPSHOT</version>
```
Comment (bshannon) [inline api/pom.xml]: This is still version 2.1 of the API.

File: api/pom.xml [hunk 4]
```diff
@@ -50,26 +63,22 @@
                 <configuration>
                     <groups>
                         <group>
-                            <title>JMS API Documentation</title>
+                            <title>Jakarta Messaging API Documentation</title>
                             <packages>
-                                javax.jms
+                                javax.messaging
```
Comment (bshannon) [inline api/pom.xml]: The package name isn't changing.

File: api/pom.xml [hunk 5]
```diff
@@ -50,26 +63,22 @@
...
                             </packages>
                         </group>
                     </groups>
                 
-                    <!-- Specifies the text to be placed at the bottom of each output file. -->
-                    <bottom>Copyright 1997-2018 Oracle America Inc. All rights reserved. Use is subject to &lt;a
-                        href="http://java.net/downloads/jms-spec/Licences/JSR 368 Java Message Service EDR 10.8.15.pdf"&gt;licence
-                        terms&lt;/a&gt; &lt;br/&gt;</bottom>
```
Comment (bshannon) [inline api/pom.xml]: 1. Do not remove the Oracle copyright notice.
2. The link to the license file should be a link to a "speclicense.html" file that's included in the javadocs.

**Comment by jgallimore** [comment]
@rmonson With regard to the ECA error, I *think* this is because this commit: https://github.com/eclipse-ee4j/jms-api/pull/230/commits/b082c9fa5848cc231f46f7617dc1e2f42d603daa doesn't have the sign off.

Not too sure if that can be changed on previous commits too easily, as it changes the commit id. You could maybe try squash merging to a new branch, and committing with the signoff and opening a new PR. I've just done 2 tests on this repo, and neither worked for whatever reason - I don't want to spam everyone, so I'm not keen to try again (unless you want me to). Happy to try and help.

File: api/src/main/java/javax/jms/DupsOKAcknowledge.java [hunk 6]
```diff
@@ -51,7 +51,7 @@
  * @see TopicListener
  * @see DupsOKAcknowledge
  *
- * @version JMS 2.1
+ * @version Jakarta Messaging 2.1
```
Comment (bshannon) [inline api/src/main/java/javax/jms/DupsOKAcknowledge.java]: There is no 2.1 version of this spec.  All the @version entries should be for 2.0.
In fact, this class should not even exist.  Ditto some of the others below.

**Comment by rmonson** [comment]
> @rmonson With regard to the ECA error, I _think_ this is because this commit: [b082c9f](https://github.com/eclipse-ee4j/jms-api/commit/b082c9fa5848cc231f46f7617dc1e2f42d603daa) doesn't have the sign off.
> 
> Not too sure if that can be changed on previous commits too easily, as it changes the commit id. You could maybe try squash merging to a new branch, and committing with the signoff and opening a new PR. I've just done 2 tests on this repo, and neither worked for whatever reason - I don't want to spam everyone, so I'm not keen to try again (unless you want me to). Happy to try and help.

I went back and rebased removing the unsigned commit and redoing it. Hopefully, that worked correctly.

File: pom.xml (hunk 1 above)
Comment (rmonson) [reply pom.xml]: fixed. Thanks!

File: README.md (hunk 2 above)
Comment (rmonson) [reply README.md]: Done!

File: pom.xml (hunk 1 above)
Comment (bshannon) [reply pom.xml]: I was off by one, this is version 2.0 of the API, matching JMS 2.0.

File: api/src/main/java/javax/jms/DupsOKAcknowledge.java (hunk 6 above)
Comment (rmonson) [reply api/src/main/java/javax/jms/DupsOKAcknowledge.java]: Good catch!  I was really thinking about that. the wrong code base must have been moved in by accident. Anyway, it should be fixed.

**Comment by dblevins** [comment]
I'm going to attempt to pull these commits onto the EE4J_8 branch.

**Comment by dblevins** [comment]
I did an `rsync` between this and the `EE4J_8` branch and we're 100% lined up on the API.  Only javadoc deltas we need remain.  Note @rmonson I did set the artifactId back to `jakarta.jms-api` as we had decided not to change those this time around.  Other than that, all looked really good.

**Pull request merged**